### ISSUE TITLE
Two urls separated by spaces will not be linkified correctly

### ISSUE THREAD

**Issue by tzehetner** [jmrware/LinkifyURL#2]
Hi,

I found a small bug:

http://www.abc.com http://www.def.com

Second url will not be linkified.

**Comment by nathan815** [comment]
Do you know of a fix for this? This was posted in 2011 and it still has that issue :/

**Comment by jmrware** [comment]
No, I wasn't aware of this issue - thanks for pointing it out. I'll look into this when I get some time - unfortunately it may be a while as I am quite busy at the moment.

**Comment by nathan815** [comment]
Okay, that's fine. I use CodeIgniter and discovered that it has a built-in [auto link helper function](https://ellislab.com/codeigniter/user-guide/helpers/url_helper.html), so I've just decided to use that instead.
